tailwind.config.js

```diff
@@ -21,33 +21,7 @@ module.exports = {
       bittersweet: "#FF6B6C",
       darkblue: "#2077F9",
       lightred: "#FF5656",
-      inherit: colors.inherit,
-      current: colors.current,
-      transparent: colors.transparent,
-      black: colors.black,
-      white: colors.white,
-      slate: colors.slate,
-      gray: colors.gray,
-      zinc: colors.zinc,
-      neutral: colors.neutral,
-      stone: colors.stone,
-      red: colors.red,
-      orange: colors.orange,
-      amber: colors.amber,
-      yellow: colors.yellow,
-      lime: colors.lime,
-      green: colors.green,
-      emerald: colors.emerald,
-      teal: colors.teal,
-      cyan: colors.cyan,
-      sky: colors.sky,
-      blue: colors.blue,
-      indigo: colors.indigo,
-      violet: colors.violet,
-      purple: colors.purple,
-      fuchsia: colors.fuchsia,
-      pink: colors.pink,
-      rose: colors.rose,
+      ...colors,
```

Comment:
`...colors,` would have to go at the beginning. 

The reason why, is because if any of the above colors are in `colors`, putting `colors` after it would override the custom color. 

So it's a good practice to always keep it on top.